transaction-ballerina/internal.bal

```diff
@@ -39,6 +40,16 @@ function startTransaction(string transactionBlockId, lang_trx:Info? prevAttempt
      }
 }
 
+function startTransactionCoordinator() {
+    http:Listener coordinatorListener = new http:Listener(coordinatorPort, { host: coordinatorHost });
+    //attach initiatorService to listener
+    error? attachInitiatorService = coordinatorListener.attach(initiatorService, "/balcoordinator/initiator");
+    // attach participant2pcService to listener
+    error? attachParticipant2pcService = coordinatorListener.attach(participant2pcService, "/balcoordinator/participant/'2pc");
```

Comment:
path has to be changed from `2pc to 2pc

Reply:
This was fixed in https://github.com/ballerina-platform/module-ballerinai-transaction/pull/33